website/docs/guides/getting_started.html.markdown

```diff
@@ -161,7 +161,7 @@ You supply the key to Terraform using the environment variable
 `GOOGLE_CLOUD_KEYFILE_JSON`, setting the value to the location of the file.
```

Comment:
Could you please replace `GOOGLE_CLOUD_KEYFILE_JSON` with  `GOOGLE_APPLICATION_CREDENTIALS` on this line as well?

Reply:
Oh, yeah sure.